requirements/aiogram.toml

```diff
@@ -0,0 +1,5 @@
+[versions]
```

Comment:
we have discussed that probably we want to have lock files for each versio, so this approach wouldn't fit